ui/app/controllers/vault/cluster/settings/configure-secret-backend.js

```diff
@@ -43,6 +42,9 @@ export default Controller.extend(CONFIG_ATTRS, {
           .catch((error) => {
             const errorMessage = error.errors ? error.errors.join('. ') : error;
             this.flashMessages.danger(errorMessage);
+          })
+          .finally(() => {
```

Comment:
the other method in this file uses `finally()` so opted for this instead of adding it to the catch block

Reply:
Makes perfect sense, since we want the loading state to be true whether the action results in a failure or success!